File: longlong-display/src/main/java/com/longlong/longlongdisplay/utils/PropertiesUtil.java
package com.longlong.longlongdisplay.utils;

import java.io.IOException;
import java.io.InputStream;
import java.util.Properties;

/**
 * @author longlong
 * @create 2020 07 09 8:40
 */
public class PropertiesUtil {

    private static InputStream is = ClassLoader.getSystemResourceAsStream("jdbc.properties");
    private static Properties properties = new Properties();
    public static String getProperty(String propertyName) throws IOException {
        properties.load(is);
        return properties.getProperty(propertyName);
    }

}

File: longlong-canal/src/main/scala/com/shenzhenhua/utils/PropertiesUtil.scala
package com.shenzhenhua.utils

import java.util.Properties

/**
 * @author longlong
 * @create 2020 07 04 6:05
 * @Describe: 该类主要提供通过传入名配置称来获取配置信息
 */

object PropertiesUtil {

  private val is = ClassLoader.getSystemResourceAsStream("conf.properties")
  private val properties = new Properties()
  properties.load(is)

  def getProperty(propertyName : String) = properties.getProperty(propertyName)

//  def main(args: Array[String]): Unit = {
//    val aaa = propertiesUtil.getProperty("bootstrap.servers")
//    println(aaa)
//  }

}
